src/lib/homestore_backend/hs_homeobject.cpp

```diff
@@ -170,7 +170,7 @@ void HSHomeObject::init_homestore() {
                 {HS_SERVICE::REPLICATION,
                  hs_format_params{.dev_type = HSDevType::Data,
                                   .size_pct = 99.0,
-                                  .num_chunks = 60000,
```

Comment:
what is the intention here? why 32768?
can we specify the chunk_size and let homestore itself determine how many chunks will be created according to the disk space?

Reply:
https://github.com/eBay/HomeObject/pull/167/files#r1571091449
pls refer this discussion why specified chunk_size is less prefered. 

Regarding the intent of move down to 32K , it is to work around the bug we are discussing in HS channel
https://ebay-eng.slack.com/archives/CAVQXEAKS/p1713889924523089